index.html

```diff
@@ -20,7 +20,7 @@ <h2>TEXT</h2>
     </div>
 
     <div id="welcome-section">
-      <h1>TEXT</h1>
+      <h1>TXT</h1>
       <p>Software developer</p>
     </div>
 
```

Comment:
@HansZizold please try to write why it should be using a section tag by explaining the benefits of that, remember it may be obvious but not for everyone